modules/npmscripts.config.js

```diff
@@ -194,6 +194,9 @@ module.exports = {
 						'clay-table': '>=2.9.0',
 						'clay-tooltip': '>=2.9.0',
 					},
+					'item-selector-taglib': {
+						'/': '>=1.0.0',
+					},
```

Comment:
Chose to put this repo-globally instead of project-locally to reduce the number of config files floating around the repo. Even though there is only one consumer, this still probably feels like the right call, because we in theory want one copy of any given dependency at any given time, and declaring them all globally kind of codifies that.

Reply:
Fair enough. 

If we could even go and make this automatic for the `/` main export (which is what most modules in DXP would only have) I think it would make this a lovable fat jar